playbook/app/pb_kits/playbook/pb_dialog/_dialog.scss

```diff
@@ -0,0 +1,111 @@
+@import "../tokens/positioning";
+@import "../tokens/colors";
+@import "../pb_card/card_mixin";
+@import "../tokens/shadows";
+
+@keyframes modalFadeIn {
+	from {
+		transform: translate3d(0, -100%, 0);
+		opacity: 0;
+	}
+	to {
+		transform: translate3d(0, 0, 0);
+		opacity: 1;
+	}
+}
+
+@keyframes modalFadeOut {
+	from {
+		transform: translate3d(0, 0, 0);
+		opacity: 1;
+	}
+	to {
+		transform: translate3d(0, -50%, 0);
+		opacity: 0;
+	}
+}
+
+@keyframes overlayFade {
+	from {
+		opacity: 0;
+		transform: translateY(0);
+	}
+	to {
+		opacity: 1;
+		transform: translateY(0);
+	}
+}
+
+@keyframes overlayFadeOut {
+	from {
+		opacity: 1;
+	}
+	to {
+		opacity: 0;
+	}
+}
+
+.dialog {
+	@include pb_card;
```

Comment:
This dialog will look a bit sharper without the border and the deepest shadow.

Reply:
Sorry, let's also play with the radius. I'm thinking 16 but you might shoot  me over a screen shot

https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/tokens/_border_radius.scss#L16